### DISCUSSION TITLE
5월 25일

### DISCUSSION THREAD

**Discussion by jinbekim** [PPiing/PPoong#42] [Backend discussions]
## 다음일정
* 29일 일요일 온라인 8시

## 할일
* 채팅 시퀀스, 구조 문서화
* 채팅 tasks 목록화  -> 금요일 저녁까지
* 일요일까지 게임 구조 문서화 및 정리 마무리

* 금요일 ~ 일요일 
* 채팅구현.

## 한일
서비스 ~ 레포까지 프로세스 및 아키텍쳐

**Comment by joohongpark** [comment]
## 아키텍쳐 협의

아키텍쳐는 다음과 같이 나눕니다.

- 컨트롤러
- 서비스
- (커스텀) 리포지토리

## 각각 담당하는 기능

### 컨트롤러

- 라우팅 + 서비스 호출
- 인자 유효성 -> 파이프

### 서비스

- 비즈니스 로직 **전부** 담당
- 여기서 에러 발생

### 커스텀 리포지토리

- 쿼리문만 담당
